extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * CuratedContent Helper
+ */
+class CuratedContentHelper {
+	const STR_ARTICLE = 'article';
+	const STR_BLOG = 'blog';
+	const STR_FILE = 'file';
+	const STR_CATEGORY = 'category';
+	const STR_EMPTY_CATEGORY = 'emptyCategory';
+	const STR_VIDEO = 'video';
+
+	public function processSections( $sections ) {
+		$processedSections = [ ];
+		if ( !empty( $sections ) && is_array( $sections ) ) {
+			foreach ( $sections as $section ) {
+				$processedSections[ ] = $this->processLogicForSection( $section );
+			}
+		}
+		return $processedSections;
+	}
+
+	private function processLogicForSection( $section ) {
+		$section['image_id'] = (int)$section['image_id']; // fallback to 0 if it's not set in request
+
+		if ( !empty( $section['items'] ) && is_array( $section['items'] ) ) {
+			foreach ( $section['items'] as &$item ) {
+				$this->fillItemInfo( $item );
+			}
+		}
+
+		return $section;
+	}
+
+	private function fillItemInfo( &$item ) {
+		$title = Title::newFromText( $item['title'] );
+		if ( !empty($title ) ) {
```

Comment:
Can you add a comment explaining why we allow titles that don't exist? Also space is missing.